src/send/Send.tsx

```diff
@@ -246,7 +251,8 @@ function Send({ route }: Props) {
         origin={TokenPickerOrigin.Send}
         onTokenSelected={onTokenSelected}
         onClose={closeCurrencyPicker}
-        tokens={isOutgoingPaymentRequest ? stableTokens : tokensWithBalance}
+        tokens={sortedTokens}
```

Comment:
Looks like this tokens change should be done separately, right?

Reply:
What do you mean? This is not changing anything actually, we were sorting inside the TokenBottomSheet, now each component can decide how to sort that list...

Reply:
Oh sorry I see! All good, I initially thought it was an unrelated change.